### PR TITLE
fix #231711: Tempo, staff text and other text honored in skipped volta

### PR DESCRIPTION
**Pull request by handrok**
See https://musescore.org/en/node/231711.

Volta stores tempo, velocity and channel at the tick before volta and sets it at the last tick of volta

velosity and channel is set inside Score::renderMidi. But tempo is set inside volta::layoutSystem because we need set tempo here because all tempos of score are set in layout. So fermata in  seconda volta works correct because fermata apply itself tempo during layouting

### CODE REVIEW

File: libmscore/volta.cpp [hunk 1]
```diff
@@ -263,6 +268,82 @@ QVariant Volta::propertyDefault(Pid propertyId) const
...
       }
 
+//---------------------------------------------------------
+//   layoutSystem
+//---------------------------------------------------------
+
+SpannerSegment * Volta::layoutSystem(System * system) {
+      SpannerSegment* voltaSegment= SLine::layoutSystem(system);
+
+      // we need set tempo in layout because all tempos of score is setted in layout
```
Comment (Jojo-Schmitz) [inline libmscore/volta.cpp]: past of set is set not setted ;-)

Comment (handrok) [reply libmscore/volta.cpp]: Oh, thanks :)

**Comment by anatoly-os** [comment]
@handrok could you please add mtest for this PR as well? It is worth appending one test per each type of the fixed elements.

File: libmscore/volta.cpp [hunk 2]
```diff
@@ -327,7 +327,7 @@ void Volta::setChannel() const {
             Staff* st = staff();
             for (int voice = 0; voice < VOICES; ++voice) {
                   int channel = st->channel(startTick, voice);
-                  st->channelList(voice)->insert(endTick, channel);
+                  st->insertIntoChannelList(voice, endTick, channel);
```
Comment (Jojo-Schmitz) [inline libmscore/volta.cpp]: this isn't about adding mtest, is it? More to fix the 1st commit's [compile error](https://travis-ci.org/musescore/MuseScore/jobs/451909849#L2330-L2333), right?

**Comment by Jojo-Schmitz** [comment]
mtest fails, see https://travis-ci.org/musescore/MuseScore/jobs/451922745#L5524

**Pull request merged**